src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -76,12 +76,20 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     void MoveSizeStart(HWND window, HMONITOR monitor, POINT const& ptScreen) noexcept
     {
         std::unique_lock writeLock(m_lock);
+        if (m_settings->GetSettings()->multiMonitorMode)

```

Comment:
let's rename `multiMonitorMode` since we are going to add support for editing zones on multi monitor (opening the editor on one monitor and able to edit zones on each individual monitor)
A better name would be `spanZonesAcrossMonitors`

Reply:
Renamed, will commit soon